launcher

```diff
@@ -224,6 +225,25 @@ restart() {
     start
 }
 
+gc() {
+    ensure_container_running
+    (
+        loginfo "Stop seafile and start garbage collector"
+        docker exec -it seafile /bin/bash -c "\
```

Comment:
It's better to create a shell script, e.g. `scripts/gc.sh`, and put the command there. The script should exit with the code of the seafile-gc.sh script. The `scripts` dir is mounted to `/scripts` inside the container.